src/widgets/RealtimeEffectPanel.cpp

```diff
@@ -0,0 +1,711 @@
+/*!********************************************************************
+
+   Audacity: A Digital Audio Editor
+
+   @file RealtimeEffectPanel.cpp
+
+   @author Vitaly Sverchinsky
+   
+**********************************************************************/
```

Comment:
Please put this new file pair directly into `src` not `src/widgets`

The sub-folder division of `src` was never really ever done on any consistent principle, whereas the coming modularization will only move files again.

The seeming intention of `src/widgets` is for reusable, and reused, windows like `Ruler` but this looks like it will only ever have one use, closely associated with ProjectWindow.